variants/STM32F3xx/F303R(6-8)T_F334R(6-8)T/variant_NUCLEO_F334R8.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2020, STMicroelectronics
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ *******************************************************************************
+ */
+#if defined(ARDUINO_NUCLEO_F334R8)
+#include "pins_arduino.h"
+
+// Digital PinName array
+const PinName digitalPin[] = {
```

Comment:
It misses several pins. Seems you only define the Arduino Uno connector. Morpho ones should also be defined.